vignettes/rticles.Rmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "Get started"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Get started}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Accessing templates
+
+```{r child="../man/fragments/usage.md"} 
+```
```

Comment:
Just curious : Why do we need to store in `man/` folder ? 

And is `man/fragments/` a convention somewhere ?

Reply:
https://www.garrickadenbuie.com/blog/dry-vignette-and-readme/

Reply:
🤔 I see. This feels weird to have that in `man/`, especially since we don't use this in any of the roxygen2 comment for help page. 

Following https://github.com/rstudio/rticles/pull/412#discussion_r672425737 I don't think we'll make a new vignette for CRAN. We only have a getting started article for the pkgdown website for now. 

So I put the file into the vignette folder.